src/swsssdk/dbconnector.py

```diff
@@ -1,20 +1,156 @@
 """
 Database connection module for SwSS
 """
-from . import _connector_map, logger
+from . import logger
 from .interface import DBInterface
-
+import os
+import json
 
 # FIXME: Convert to metaclasses when Py2 support is removed. Metaclasses have unique interfaces to Python2/Python3.
 
+class SonicDBConfig(object):
+    SONIC_DB_CONFIG_FILE = "/var/run/redis/sonic-db/database_config.json"
+    _sonic_db_config_init = False
+    _sonic_db_config = {}
+
+    @staticmethod
+    def load_sonic_db_config(sonic_db_file_path=SONIC_DB_CONFIG_FILE):
+        """
+        Get multiple database config from the database_config.json
+        """
+        if SonicDBConfig._sonic_db_config_init == True:
+            return
+
+        try:
+            if os.path.isfile(sonic_db_file_path) == False:
+                sonic_db_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'config', 'database_config.json')
+            with open(sonic_db_file_path, "r") as read_file:
+                SonicDBConfig._sonic_db_config = json.load(read_file)
+        except (OSError, IOError):
+            msg = "Could not open sonic database config file '{}'".format(sonic_db_file_path)
+            logger.exception(msg)
+            raise RuntimeError(msg)
+        SonicDBConfig._sonic_db_config_init = True
+
+    @staticmethod
+    def get_sonic_db_list():
+        if SonicDBConfig._sonic_db_config_init == False:
+            SonicDBConfig.load_sonic_db_config()
+        return SonicDBConfig._sonic_db_config["DATABASES"].keys()
+
+    @staticmethod
+    def get_sonic_db_socket(db_name):
+        if SonicDBConfig._sonic_db_config_init == False:
+            SonicDBConfig.load_sonic_db_config()
+        inst_name = SonicDBConfig._sonic_db_config["DATABASES"][db_name]["instance"]
```

Comment:
>db_name [](start = 64, length = 7)

Should we handle index out of range explicitly?
1. if db_name does not exist, it's invalid argument
2. if inst_name does not exist, it's invalid config.

Reply:
added db_name_validation() and inst_name_validation(), if invalid , raise a error